q2_matchmaker/tests/test_stan.py

```diff
@@ -47,6 +47,37 @@ def test_cc(self):
         az.rhat(inf, var_names=nb.param_names)
         az.ess(inf, var_names=nb.param_names)
 
+    def test_cc_fit(self):
+        biom_table = Table(self.table.values.T,
+                           list(self.table.columns),
+                           list(self.table.index))
+        nb = NegativeBinomialCaseControl(
+            table=biom_table,
+            matching_column="reps",
+            status_column="diff",
+            metadata=self.metadata,
+            reference_status='1',
+            mu_scale=1,
+            sigma_scale=.1,
+            disp_scale=0.01,
+            # priors specific to the simulation
+            control_loc=-6,
+            control_scale=3,
+            chains=1,
+            seed=42)
```

Comment:
Just in case someone comes here and wonders what's going on here - we are forcing a negative binomial model to behave like a multinomial model.  Is it ideal? No, but it helps us provide a ballpark perspective to see if the model is performing as expected.